xchange-lgo/src/main/java/org/knowm/xchange/lgo/dto/marketdata/LgoPriceHistory.java

```diff
@@ -0,0 +1,47 @@
+package org.knowm.xchange.lgo.dto.marketdata;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.util.*;
+import org.knowm.xchange.utils.DateUtils;
+
+public class LgoPriceHistory {
+
+  private final List<LgoCandlestick> prices;
+
+  private LgoPriceHistory(List<LgoCandlestick> prices) {
+    this.prices = prices;
+  }
+
+  public static LgoPriceHistory fromRawValues(List<List<Object>> rawPriceHistory)
```

Comment:
use `@JsonFormat(shape = Shape.ARRAY)` with `LgoCandlestick` instead of doing manual parsing

Reply:
done. Nice Jackson feature that I wasn't aware of, thank you